src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -197,14 +199,14 @@ public void ErrorListInfoBarController_Refresh_ActiveSolutionBound_NotFullyLoade
 
             // Assert
             // Action should have been queued because the context is not active
-            uiContext.Verify(x => x.WhenActivated(testSubject.ProcessSolutionBinding), Times.Once());
+            uiContext.Verify(x => x.WhenActivated(It.IsAny<Action>()), Times.Once());
```

Comment:
The synchronous product method is now private. This was the only place that used it, so I decided to make the assertion weaker rather than making the product method more visible.